src/openzaak/components/catalogi/api/viewsets/mixins.py

```diff
@@ -55,29 +56,26 @@ class ConceptMixin(ConceptPublishMixin, ConceptDestroyMixin, ConceptFilterMixin)
     pass
 
 
-class ZaakTypeConceptCreateMixin:
-    def perform_create(self, serializer):
-        zaaktype = serializer.validated_data["zaaktype"]
-        if not zaaktype.concept:
-            msg = _("Creating a related object to non-concept object is forbidden")
-            raise PermissionDenied(detail=msg)
-
-        super().perform_create(serializer)
-
-
 class ZaakTypeConceptDestroyMixin(ConceptDestroyMixin):
     def get_concept(self, instance):
         return instance.zaaktype.concept
 
+    def perform_destroy(self, instance):
```

Comment:
I don't quite understand why to rewrite this method, because logic remains the same. An error code can be put into class variable I think.
Or if you want to create your own `perform_destroy` the inheritance of `ConceptDestroyMixin` is not needed. In current realization you will invoke `ConceptDestroyMixin.perform_destroy` method which will never raise an error.